src/custom-elements/src/components/drawer/drawer.tsx

```diff
@@ -214,10 +214,17 @@ export class Drawer {
 
   private _documentClickHandler = (ev): void => {
     const drawerElement = this.el.querySelector('.chi-drawer');
+    const drawerCloseButton = !this.nonClosable ? drawerElement.querySelector('button.-close') : null;
```

Comment:
For not using double negation:
```suggestion
    const drawerCloseButton = this.nonClosable ? null : drawerElement.querySelector('button.-close');
```

Reply:
Makes sense. Thanks!